Test of pullrequest review

LukaDevic opened the pull request:
Test of pullrequest review...

LukaDevic committed:
added new feature (Add)

LukaDevic committed:
Merge pull request #1 from LukaDevic/feature/newFeature

added new feature (Add)

LukaDevic committed:
added ...

marisks commented on GitCommandsTestWithConsoleApp/obj/Debug/GitCommandsTestWithConsoleApp.csproj.CoreCompileInputs.cache:
```diff
@@ -1 +1 @@
-8da5fbcb33b595602d5f7d2c590d0554bce8e149
+044f68d0a95dd498c0690d17254eed5dc4ad4bd2
```
Debug folder should not be included in a git repository. Add it to the .gitignore file.

LukaDevic replied:
Added it to .gitignore in commit # 387e953. Also added .vs folder to .gitignore folder in the same commit and removed it from github according to example in commit # 32cb6ac. Thank you for review and examples. :)

LukaDevic committed:
created .gitignore file and added debug folder to it

LukaDevic committed:
created .gitignore file and added debug folder to it

LukaDevic committed:
created .gitignore file and added debug folder to it

LukaDevic committed:
corected .gitignore file

LukaDevic committed:
corected .gitignore again

marisks commented:
You can take this .gitignore as an example: https://github.com/marisks/examples/blob/master/.gitignore

Also, remove `.vs` folder. After doing it, you have to remove those files from the git. Adding those to .gitignore doesn't remove those from the git repository.

You can remove files and folders as described here: https://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste

LukaDevic committed:
finally made the gitignore in the right way :D

LukaDevic committed:
.gitignore changed

LukaDevic committed:
updated git ignore acording to example, added .vs folder to git ignore

LukaDevic committed:
removed .vs folder acording to example

marisks commented:
Now the project looks good but you have to rebase it on top of master. As you see, there are `Conflicting files`. It means that there are changes in master and in your branch of the same files.

LukaDevic committed:
rebased develop branch with master branch

LukaDevic commented:
> Now the project looks good but you have to rebase it on top of master. As you see, there are `Conflicting files`. It means that there are changes in master and in your branch of the same files.

I have rebased devlop branch with master branch locally and updated remote devlop branch trough merge with local develop branch. There are no conflicting files anymore. :)
